internal/drivers/azure/driver.go

```diff
@@ -189,10 +192,11 @@ func (c *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (in
 				},
 			},
 			OSProfile: &armcompute.OSProfile{ //
-				ComputerName:  to.Ptr("vm-runner"),
-				AdminUsername: to.Ptr(c.username),
-				AdminPassword: to.Ptr(c.password),
-				CustomData:    to.Ptr(uData),
+				ComputerName:             to.Ptr("vm-runner"),
+				AdminUsername:            to.Ptr(c.username),
+				AdminPassword:            to.Ptr(c.password),
+				CustomData:               to.Ptr(uData),
+				AllowExtensionOperations: to.Ptr(true),
```

Comment:
does this cause any overhead? If so maybe we should only enable for windows and not linux

Reply:
none